drivers/timers/Make.defs

```diff
@@ -28,6 +28,12 @@ ifeq ($(CONFIG_WATCHDOG),y)
   TMRVPATH = :timers
 endif
 
+ifeq ($(CONFIG_WATCHDOG_TIMEOUT_NOTIFIER),y)
+  CSRCS += wdt_notifier.c
```

Comment:
```suggestion
  CSRCS += watchdog_otifier.c
```

Reply:
I have changed all the wdt_/WDT_ prefixes in this commit to watchdog_/WATCHDOG_